docs/en/datasets/det.md

```diff
@@ -870,3 +871,60 @@ inconsistency results in false examples in the training set. Therefore, users sh
   │   ├── instances_training.json
   │   └── instances_val.json (optional)
   ```
+
+## HierText
+
+- Step1 (optional): Install [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html). Here is a brief installation procedure, if you encounter problems, please check the official website.
+  ```bash
+  curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+  unzip awscliv2.zip
+  sudo ./aws/install
+  ./aws/install -i /usr/local/aws-cli -b /usr/local/bin
+  !aws configure
+  # this command will require you to input keys, you can skip them except
+  # for the Default region name
+  # AWS Access Key ID [None]:
+  # AWS Secret Access Key [None]:
+  # Default region name [None]: us-east-1
+  # Default output format [None]
```

Comment:
Since there are also other datasets that use AWS CLI, we may create a new section after the Overview section and link all these steps to that section.

Reply:
Good idea.